api/can_api_v2_definition.py

```diff
@@ -61,6 +61,13 @@ class NonCovidPatientsMethod(enum.Enum):
 
 
 class DemographicDistributions(base_model.APIBaseModel):
+    """Distributions of demographic data.
+
+    Note that because different regions may have different demographic distributions for
+    the same field.  For instance, how age is collected is not uniform.
+
+    The data provided matches the source distributions.
```

Comment:
We may filter the values so the values may not match our upstream sources exactly. How about "Distributions match those provided by upstream data sources."